Documentation/operations/system_requirements.rst

```diff
@@ -318,6 +318,51 @@ Port Range / Protocol    Description
 9876/tcp                 cilium-agent health status API
 ======================== ==========================================
 
+.. _admin_mount_bpffs:
+
+Mounted eBPF filesystem
+=======================
+
+.. Note::
+
+        Some distributions mount the bpf filesystem automatically. Check if the
+        bpf filesystem is mounted by running the command.
+
+        .. code-block:: shell-session
+
+                  mount | grep /sys/fs/bpf
+                  # if present should output, e.g. "none on /sys/fs/bpf type bpf"...
+
+This step is **required for production** environments but optional for testing
+and development. It allows the ``cilium-agent`` to pin eBPF resources to a
+persistent filesystem and make them persistent across restarts of the agent.
+If the eBPF filesystem is not mounted in the host filesystem, Cilium will
+automatically mount the filesystem but it will be unmounted and re-mounted when
+the Cilium pod is restarted. This in turn will cause eBPF resources to be
+removed which impacts cross-node forwarding until the eBPF filesystem is
+remounted and Cilium restarts. Mounting the eBPF filesystem in the host mount
```

Comment:
I am not sure remount will fix the issue, since the calls map has already been deleted.

Reply:
It's more the `Cilium restarts` bit at the end that should fix it. Maybe it should just be:

> This in turn will cause eBPF resources to be
> removed which impacts cross-node forwarding until Cilium restarts.